kernelci.org/content/en/docs/instances/chromeos/chromeos_changelog.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "ChromeOS image changelog"
+date: 2023-02-03
+weight: 4
+---
+
+This file tracks divergences between the KernelCI built ChromiumOS images and upstream ChromiumOS as well as any changes between image relases.
+
+Divergences can be considered tech-debt and in the long run need to be kept under control and minimized, therefore this chanelog should reflect its evolution from version to version.
+
+### Fetching latest images
+
+It is recommended to use the latest published image versions for each board from [this list](https://storage.chromeos.kernelci.org/images/rootfs/chromeos/).
+
+The latest version can be found either from the directory date name (e.g. `chromiumos-asurada/20230208.0`) or by the `distro_version` field in the `manifest.json` file, where for e.g. R106 is greater than R100.
+
+## Release 106
+
+### Supported boards
+
+Direct links for each supported board in this release are provided here for convenience:
```

Comment:
For some reason this didn't render as a list:
https://static.staging.kernelci.org/docs/instances/chromeos/chromeos_changelog/

Have you tried running an instance of the website locally?  Maybe running `markdown` locally can also show the problem.  If you can't reproduce it I can take a look as I have a local instance all set up.

Reply:
I'm in favour of having an actual list.  If it really gets too long over time we can maybe have several columns or something like that.  But also I'm in favour of making what looks like a list in Markdown actually render as a list in HTML otherwise it's quite confusing for anyone editing the file.